js/build.js

```diff
@@ -133,7 +136,16 @@
   });
 
   // Cleanup unused Collapsible Ends
-  $(Collapsible.SELECTORS.collapsibleEnd).parents(Collapsible.SELECTORS.widget).remove();
+  $(Collapsible.SELECTORS.collapsibleEnd).parents(Collapsible.SELECTORS.widget)
+    .each(function () {
```

Comment:
This executes `.remove()` once instead of multiple times.

```suggestion
    .not(_.map(nestingWidgets, function (name) {
      return '[data-widget-package="' + name + '"]';
    }).join(', ')).remove();
```

Reply:
Updated with suggestion!